crates/vm2/src/precompiles/legacy.rs

```diff
@@ -311,7 +335,7 @@ mod tests {
 
         prop_assert_eq!(output.len, 2);
         let expected_address = key_to_address(signing_key.verifying_key());
-        let [is_success, address] = output.buffer;
+        let [is_success, address, _address] = output.buffer;
```

Comment:
Nit: Please use `_` for the third word here and below; it doesn't have address / "is_verified" semantics.